src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -229,56 +233,45 @@ const TokensMoveDialogForm = ({
         />
       </DialogSection>
       <DialogSection>
-        <Select options={domainOptions} label={MSG.from} name="fromDomain" />
-        {!!tokenAddress && (
-          <div className={styles.domainPotBalance}>
-            <FormattedMessage
-              {...MSG.domainTokenAmount}
-              values={{
-                amount: (
-                  <Numeral
-                    appearance={{
-                      size: 'small',
-                      theme: 'grey',
-                    }}
-                    value={fromDomainTokenBalance || 0}
-                    unit={getTokenDecimalsWithFallback(
-                      selectedToken && selectedToken.decimals,
-                    )}
-                    truncate={3}
-                  />
-                ),
-                symbol: (selectedToken && selectedToken.symbol) || '???',
-              }}
-            />
-          </div>
-        )}
-      </DialogSection>
-      <DialogSection>
-        <Select options={domainOptions} label={MSG.to} name="toDomain" />
-        {values.toDomain !== undefined && !!values.tokenAddress && (
-          <div className={styles.domainPotBalance}>
-            <FormattedMessage
-              {...MSG.domainTokenAmount}
-              values={{
-                amount: (
-                  <Numeral
-                    appearance={{
-                      size: 'small',
-                      theme: 'grey',
-                    }}
-                    value={toDomainTokenBalance || 0}
-                    unit={getTokenDecimalsWithFallback(
-                      selectedToken && selectedToken.decimals,
-                    )}
-                    truncate={3}
-                  />
-                ),
-                symbol: (selectedToken && selectedToken.symbol) || '???',
-              }}
+        <div className={styles.domainSelects}>
+          <div>
+            <Select
+              options={domainOptions}
+              label={MSG.from}
+              name="fromDomain"
             />
+            {!!tokenAddress && (
+              <div className={styles.domainPotBalance}>
+                <FormattedMessage
```

Comment:
In general all the fonts on this modal are too small. Please follow the spec on figma for proper font sizing.

For example, in this case:
![Screenshot from 2020-11-26 14-43-34](https://user-images.githubusercontent.com/1193222/100354022-22f7eb80-2ff8-11eb-8cc0-3ab1264120af.png)

Your
![Screenshot from 2020-11-26 14-43-53](https://user-images.githubusercontent.com/1193222/100354060-2ee3ad80-2ff8-11eb-9729-281ab2f9b416.png)
 font size is `12px` while the spec requires it to be `13px`